docs/user/dashboard/dashboard.asciidoc

```diff
@@ -85,13 +86,15 @@ add an existing panel from the *Visualize Library*, add a table of live streamin
 [[create-panels-with-lens]]
 ==== Create panels
 
-To create panels, use one of the editors, then add the panel to the dashboard.
+To create panels, choose the panel type you want to add, configure the panel, then add the panel to the dashboard. 
+
+. On the dashboard, choose one of the following options:
 
-. On the dashboard, click *Create panel*.
+* Click *Create visualization* to open *Lens*, the recommended visualization editor. 
```

Comment:
There's a potential gotcha in here where, by default, visualizations created from a Dashoard are not added to the library. Is the Lens save modal (with option to 'Add to library') or library concept covered elsewhere?

The main thing I'm attempting to cover is where you try to 'Add from library' (open the flyout in Dashboard) and you can't find the visualization because it was never added to the library.

Reply:
> There's a potential gotcha in here where, by default, visualizations created from a Dashboard are not added to the library. Is the Lens save modal (with option to 'Add to library') or library concept covered elsewhere?
> 
> The main thing I'm attempting to cover is where you try to 'Add from library' (open the flyout in Dashboard) and you can't find the visualization because it was never added to the library.

@ryankeairns yes! We cover this in lines 99-107:

. To add the panel to the dashboard, choose one of the following options:

* To add the panel to the dashboard without saving to the *Visualize Library*, click *Save and return*. 
+
To add a title to the panel, click *No Title*, enter the *Panel title*, then click *Save*.

* To save the panel to the *Visualize Library*, click *Save to Library*, configure the options, then click *Save and return*.
+
When panels are saved in the *Visualize Library*, image:dashboard/images/visualize-library-icon.png[Visualize Library icon] appears in the header.

Here is what this section looks like in the docs:
<img width="752" alt="Screen Shot 2021-05-20 at 9 16 27 AM" src="https://user-images.githubusercontent.com/38990715/118994629-1d96b080-b94c-11eb-9be9-a3c475a13025.png">


If you didn't immediately find this information, I'm now thinking that I can improve this section...maybe with images.

Reply:
Ah ok, thanks for pointing that out. The library concept is global in nature 🤔  ... also, it's most troublesome when you click the 'Add from library' button when in Dashboard edit mode. Is there a place in the docs for this particular interaction? What I'm ultimately driving toward is a note of sorts that says "Not seeing your x? Make sure it was added to the visualization library. You can do this by..."

Reply:
The library concept is global in nature, but with the dashboard-first initiative, the Visualize content was removed and incorporated into the Dashboard section. 

I have an idea! I'm going to put a Visualize Library section in the Kibana concepts section. This is where we are housing our global Kibana concepts.